### PR TITLE
test: document empty input behavior in get_value_vars_from_user_vars

### PR DESCRIPTION
**Pull request by jyotihotwani25**
## Description
This PR adds a test documenting that `get_value_vars_from_user_vars([], model)` returns an empty list when given empty input.

The test reflects the current behavior and improves coverage for an edge case without changing any functionality.

## Related Issue

- [ ] Closes #
- [ ] Related to #

## Checklist

- [x] Checked that [the pre-commit linting/style checks pass](https://docs.pymc.io/en/latest/contributing/python_style.html)
- [x] Included tests that prove the fix is effective or that the new feature works
- [ ] Added necessary documentation (docstrings and/or example notebooks)
- [ ] If you are a pro: each commit corresponds to a [relevant logical change](https://wiki.openstack.org/wiki/GitCommitMessages#Structural_split_of_changes)

## Type of change

- [ ] New feature / enhancement
- [ ] Bug fix
- [ ] Documentation
- [x] Maintenance
- [ ] Other (please specify):

### CODE REVIEW

**Comment by ricardoV94** [comment]
Why do you feel this is needed?

File: tests/test_util.py [hunk 1]
```diff
@@ -241,6 +241,9 @@ def test_get_value_vars_from_user_vars():
         y2 = pm.Normal("y2", mu=0, sigma=1)
         det2 = pm.Deterministic("det2", x2 + y2)
 
+    with pm.Model() as model:
+        assert get_value_vars_from_user_vars([], model) == []
```
Comment (ricardoV94) [inline tests/test_util.py]: You can add it in the first block of checks from the first model. It doesn't need to be tested with an empty model?

Comment (jyotihotwani25) [reply tests/test_util.py]: The reason for adding this check is to explicitly document the current contract that empty input is a valid case and returns an empty list, rather than failing implicitly or raising an error. Without an explicit test, this behavior could
easily change unintentionally during refactors without being caught.

I agree that the behavior itself does not depend on model contents, and placing the assertion alongside the existing `model1` checks makes the intent clearer. I’ve moved it there.

I’m still getting familiar with the PyMC codebase, if there are particular parts you’d recommend focusing on early when exploring the codebase, I’d appreciate any pointers.
Thanks for the review.

**Review by ricardoV94 — APPROVED** [review]

**Comment by ricardoV94** [comment]
I'm approving this one @jyotihotwani25 , but I feel it's a code-change looking for a reason rather than the other way around. Python is loosely typed and any function that accepts/returns containers will have empty edge case. I don't believe we should test it all just "because". I suggest looking for code changes that are meaningful to the users for your next steps (if you so wish to proceed), or actual bug-fixes.

Rule-of-thumb: if there's no issue open for it, it's probably not a good first issue

**Comment by jyotihotwani25** [comment]
Thanks for the approval and the feedback. I’ll focus future contributions on more clearly user-facing and impactful changes.
I Appreciate the guidance.
